redaxo/src/core/lib/response.php

```diff
@@ -387,6 +387,79 @@ protected static function sendGzip($content)
         return $content;
     }
 
+    // method inspired by https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Cookie.php
+    /**
+     * @param string                        $name     The name of the cookie
+     * @param string|null                   $value    The value of the cookie
+     * @param int|string|\DateTimeInterface $expire   The time the cookie expires
+     * @param string                        $path     The path on the server in which the cookie will be available on
+     * @param string|null                   $domain   The domain that the cookie is available to
+     * @param bool                          $secure   Whether the cookie should only be transmitted over a secure HTTPS connection from the client
+     * @param bool                          $httpOnly Whether the cookie will be made accessible only through the HTTP protocol
+     * @param string|null                   $sameSite Whether the cookie will be available for cross-site requests
+     * @param bool                          $raw      Whether the cookie value should be sent with no url encoding
+     *
+     * @throws \InvalidArgumentException
+     */
+    public static function sendCookie($name, $value = null, $expire = 0, $path = '/', $domain = null, $secure = false, $httpOnly = true, $sameSite = null, $raw = false)
```

Comment:
Ich überlege gerade, ob man nicht eher die hinteren Parameter zusammenfassen sollte in ein `$options`-Array, sodass man leichter auch nur eine Option ganz hinten setzen kann, und um diese `..., false, true, ...`-Dinger zu vermeiden

Reply:
mit diesen gedanken hatte ich auch gespielt. aber dann aus faulheit an symfony orientiert ;)

Reply:
Oder `rex_response::sendCookie(rex_cookie::factory($name, $value)->expire($x)->path(...)...)`?

Reply:
Cookie setzen ist ja auch nichts, was man dauernd macht. von daher ist es für mich auch so ok, wie es jetzt ist.